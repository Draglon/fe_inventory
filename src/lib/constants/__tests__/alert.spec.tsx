import * as alert from "../alert";

it("alert constants match snapshot", () => {
  expect(alert).toMatchSnapshot();
});
