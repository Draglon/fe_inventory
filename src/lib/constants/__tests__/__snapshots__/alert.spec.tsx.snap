// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`alert constants match snapshot 1`] = `
{
  "ALERT_KINDS": {
    "error": "error",
    "info": "info",
    "success": "success",
    "warning": "warning",
  },
}
`;
